pkg/apis/extensions/v1beta1/conversion.go

```diff
@@ -514,3 +515,7 @@ func Convert_networking_NetworkPolicyList_To_v1beta1_NetworkPolicyList(in *netwo
 func Convert_extensions_PodSecurityPolicySpec_To_v1beta1_PodSecurityPolicySpec(in *extensions.PodSecurityPolicySpec, out *extensionsv1beta1.PodSecurityPolicySpec, s conversion.Scope) error {
 	return autoConvert_extensions_PodSecurityPolicySpec_To_v1beta1_PodSecurityPolicySpec(in, out, s)
 }
+
+func Convert_v1beta1_PodSecurityPolicySpec_To_extensions_PodSecurityPolicySpec(in *extensionsv1beta1.PodSecurityPolicySpec, out *extensions.PodSecurityPolicySpec, s conversion.Scope) error {
```

Comment:
why is this added as a manual conversion?

Reply:
frankly i dont know how to auto generate it , i ran all hack/updatexxx and didn't see it generated. if you have suggestions let me know

Reply:
@krmayankk what is the output from the generator when you tried to generate the conversions?